examples/advanced_demo.py

```diff
@@ -358,8 +358,8 @@ def init_gui_stm32(self):
         lcd.init(w=hres, h=vres)
         self.disp_drv = lv.display_create(hres, vres)
         self.disp_drv.set_flush_cb(lcd.flush)
-        buf1_1 = bytearray(hres * 50 * lv.color_t.__SIZE__)
-        buf1_2 = bytearray(hres * 50 * lv.color_t.__SIZE__)
+        buf1_1 = bytearray(hres * 50 * lv.COLOR_DEPTH // 8)
```

Comment:
`lv.color_format_get_size(color_format)`

Reply:
for this one I believe there is a function called lv.color_format_get_bpp

Reply:
Fixed